components/user_settings/display/custom_theme_chooser.jsx

```diff
@@ -136,11 +136,6 @@ class CustomThemeChooser extends React.Component {
                 [settingId]: color,
             };
 
-            // For backwards compatability
-            if (settingId === 'mentionBg') {
```

Comment:
why are you removing this? it should be there for backwards compatibility for those users who have the prop name as `mentionBj` in their theme settings if not done that way their theme might break